src/components/InfoModal.tsx

```diff
@@ -89,19 +89,52 @@ function InfoModal(): JSX.Element {
     concentration: concName,
     grad_year: grad,
     auth0_id: id,
+    enroll: enrollYear,
+    planned_grad: plannedGradYear,
   } = info
 
   // Changing information in modal
   const [name, setName] = useState(firstName)
   const nameRef = useRef('')
 
-  const [school, setSchool] = useState(schoolName)
+  const [school, setSchool] = useState(schoolDict[schoolName])
 
-  const [major, setMajor] = useState(majorName)
+  const [major, setMajor] = useState(majorDict[majorName])
 
   const [concentration, setConcentration] = useState(concName)
 
-  const [gradYear, setGradYear] = useState(String(grad))
+  const [enroll, setEnroll] = useState(String(enrollYear))
+
+  const [plannedGrad, setPlannedGrad] = useState(plannedGradYear)
+
+  const updateGradDates = (start: number): { value: string }[] => {
+    return [
+      {
```

Comment:
Nice, I like that it's capped at 4 years. In the future, we may consider more but this is good for now.